kedro/templates/project/hooks/post_gen_project.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
```

Comment:
I'd recommend using [Pathlib](https://docs.python.org/3/library/pathlib.html) for file path handling. We use this in most places in Kedro and it's more robust across operating systems.